src/api/handlers/messages/getMessages.js

```diff
@@ -1,26 +1,15 @@
-export default (req, res) => {
-    const { appointmentId } = req.params;
+const { db } = require("../../../../knex");
```

Comment:
Favour import over require
```suggestion
import { db } from "../../../../knex";
```

Reply:
fixed!